### PR TITLE
deploy-20231102

### PR DESCRIPTION
**Pull request by FXF48-Mila**

### CODE REVIEW

**Comment by FXF48-Mila** [comment]
Hi @aditya-uw, there are two things I want to mention and would appreciate your advice. 1) The Audiomoth K we recovered on Thursday showed an abnormal ending voltage (which I have documented in the comment). It ended at a voltage of 2.935, which does not pass the pytest (because the current pytest limits the ending voltage to be a number between 3-4). In this case, should I change the limit boundary of the pytest? 2) I remember the scribe for Nov 2 was Varun, but the valid people in our pytest didn't have Varun's initial. Now I've written scribe as you. Should I add Varun's initial into pytest and change the scribe to Varun? In this case, should his initial be "VK"?

**Comment by FXF48-Mila** [comment]
Hi @aditya-uw, can you check the PR?

File: field_records/ubna_2023.csv [hunk 1]
```diff
@@ -126,7 +126,11 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
...
-2023-10-26T11:24:00 , (DATE-TIME) , L , 012 , Telephone Field , 47° 39' 25.410'' N , 122° 17' 29.220'' W , 192000 , Medium , None , None , 4.154 , (VOLTAGE) , MS , AK , MS , (RECOVER-DATE) , Panasonic Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
-2023-10-26T11:24:00 , (DATE-TIME) , I , 007 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.718'' W , 192000 , Medium , None , None , 4.211 , (VOLTAGE) , MS , AK , MS , (RECOVER-DATE) , Panasonic Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
+2023-10-26T11:23:00 , 2023-11-02T17:50:00 , J , 002 , E18 Bridge , 47° 39' 21.198'' N , 122° 17' 48.612'' W , 192000 , Medium , None , None , 4.360 , 3.648 , MS , AK , MS , recover-20231102 , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
+2023-10-26T11:23:00 , 2023-11-02T18:14:00 , K , 010 , Central Pond , 47° 39' 15.360'' N , 122° 17' 34.500'' W , 192000 , Medium , None , None , 4.152 , 2.935 , MS , AK , MS , recover-20231102 , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF; Unusual ending voltage; the Audiomoth's black wire was broken;
```
Comment (aditya-uw) [inline field_records/ubna_2023.csv:130]:
```suggestion
2023-10-26T11:23:00 , 2023-11-02T18:14:00 , K , 010 , Central Pond , 47° 39' 15.360'' N , 122° 17' 34.500'' W , 192000 , Medium , None , None , 4.152 , 2.935 , MS , AK , MS , recover-20231102 , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF; Unusual ending voltage; Audiomoth's 2nd battery pack became disconnected; Connection needed to be re-soldered;
```

**Comment by aditya-uw** [comment]
Okay yeah @FXF48-Mila, I just made a few small suggestions. You can commit the suggestion as I put it.

I also saw that your regex pattern was like (2|3|4) so I instead made it [2-4] which also works. I did not know if it would work so I changed this myself. Once you commit my suggestion, feel free to squash and merge!

**Pull request merged**